clippy_dev/src/stderr_length_check.rs

```diff
@@ -2,7 +2,7 @@ use std::ffi::OsStr;
 use walkdir::WalkDir;
 
 use std::fs::File;
-use std::io::prelude::*;
+use std::io::prelude::Read;
```

Comment:
For some reason `BufRead` was also imported after fixing it with rustfix. Here was the second `unused_imports` warning.